rpmlint/ReadelfParser.py

```diff
@@ -0,0 +1,151 @@
+import re
```

Comment:
Please do not camelcase filenames :)

Reply:
Do you mean to rename it to `ReadElfParser.py`?

Reply:
`rpmlint/readelfparser.py`